tests/lib/deno_test.generated.js

```diff
@@ -0,0 +1,186 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
I included the generated file here so we can watch how it changes over time or catch any regressions more clearly.